vite.config.ts

```diff
@@ -4,8 +4,7 @@ import react from '@vitejs/plugin-react';
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
-  server: {
-    host: '127.0.0.1',
-    port: 8080
-  }
-});
+   server: {
+    port: 8080,
+  },
+})
```

Comment:
we want to keep `host: '127.0.0.1'` so that the dev environment starts up in `127.0.0.1` instead of `localhost`!